### PR TITLE
create linhle199611.md

### PR DESCRIPTION
**Pull request by linhle199611**
https://raw.githack.com/linhle199611/linhle199611.github.io/linhle/#!pages/vi/profiles/linhle199611.md

### CODE REVIEW

**Review by kjong — CHANGES_REQUESTED** [review]
Please try and use markdown instead of html (eg. `#` instead of `<h1>`).
One of the goals of this exercise is to become more familiar with markdown.

File: pages/vi/profiles/linhle199611.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h1>Linh Le </h1>
+<h2>San Jose, California / Pacific Standard Time </h2>
+<h2>OS: Ubuntu 18.04.2 LTS </h2>
+
+Language: `Java` `C++`
+
+My name is Linh Le. I am a senior student at San Jose State University. I am very excited to be a part of this project and hopefully I have a chance to meet everyone who help to build this project strong and amazing. <br>
+
+> - My Facebook: [Linh](https://www.facebook.com/linhlinhlinhlinh.lelelele) <br>
```
Comment (dogi) [inline pages/vi/profiles/linhle199611.md:9]: @linhle199611 please also do not use `<br>`

**Review by dogi — COMMENTED** [review]
good start :)

**Comment by dogi** [comment]
@linhle199611 why would you not just change it to markdown?

**Pull request closed without merging**